docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -536,7 +535,7 @@ A sample denotes a single pair of inputs/labels.
 
 2. Batch
 
-A batch is a bundle of samples which is fed to a single iteration. If a batch is of size 5, the batch layout, where `/*` denotes nth sample will be:
+A batch is a bundle of samples which is fed to a single iteration. If a batch is of size 5, the batch layout, where `/*` denotes nth sample will be as follows:
```

Comment:
**Line 538-546:** Proper indentation should be maintained under the bullet points.

Reply:
Updated